packages/e2e-lambdatest/package.json

```diff
@@ -1,19 +0,0 @@
-{
-  "name": "@zk-email/e2e-lambdatest",
```

Comment:
Wait so where did this package go

Reply:
Moved to twitter-repo. They are twitter specific tests.